en/api/jdbc.rst

```diff
@@ -244,6 +245,9 @@ The **getConnection** method returns the **Connection** object and it is used to
     --connection URL string when hold_cursor property specified for cursor holdability
     URL=jdbc:CUBRID:192.168.0.1:33000:demodb:public::?hold_cursor=true    
 
+    --connection URL string when oracleStyleNumberReturn property is specified for remove decimal zeros from the result value 
```

Comment:
--connection URL string when oracleStyleNumberReturn property is specified to remove trailing zeros after the decimal point from the result value

Reply:
thanks for the comment I'll fix it.